doc/e2ee.md

```diff
@@ -87,6 +87,8 @@ between the counter and the trailing bit:
    +^+-------------------------------------------------------+ +
    | |                 Authentication Tag                    | |
    | +---------------------------------------+-+-+-+-+-+-+-+-+ |
+   | | additional authentication tags        |number of tags | |
```

Comment:
This is the number of additional tags, i.e. not including the authentication tag